src/Data/Lens/Traversal.purs

```diff
@@ -1,12 +1,30 @@
--- | This module defines functions for working with traversals.
+-- | `Traversal` is an optic that focuses on zero or more functor values. An
```

Comment:
Not necessarily a `Functor`, but something isomorphic to a `Traversable` functor (just a `Functor` gives a `Setter` actually). For example, I can traverse the chars in a string.

Reply:
I'll just say "Traversable functor". My goal here is to allow people to use `Traversal` without having to first know `Traversable` (not in all cases, but the simple ones).

Reply:
The String/Char example does not involve a `Traversable` functor though. I think simply removing the word `functor` would be enough.